scripts/tpg_find_overactive_channels.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+"""
+For a given TPStream, plot the channel histogram and print
+the channels that are above a given limit and Tuckey's fence.
+"""
+
+
+from trgtools import TPReader
+
+import click
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+import os
+
+
+# As of 05-13-2024.
+DETECTOR_CHANNELMAPS = {
+            "HD": os.path.join(os.environ["DETCHANNELMAPS_SHARE"], "config/pd2hd/PD2HDChannelMap_v6.txt"),
+            "VD": os.path.join(os.environ["DETCHANNELMAPS_SHARE"], "config/pd2vd/PD2VDBottomTPCChannelMap_v1.txt"),
+            "VDCB": os.path.join(os.environ["DETCHANNELMAPS_SHARE"], "config/vdcoldbox/vdcbce_chanmap_v4.txt")
+        }
+
+# Colors.
+BLUE = "#63ACBE"
+RED  = "#EE442F"
+
+
+def get_detector_element_channel_hist_map(map_name: str) -> dict[str, int]:
+    """
+    Get a map with the detector elements as the key and the channels (as histogram edges)
+    as the value.
+
+    Parameters:
+        map_name (str): Channel map name to load from.
+
+    Returns a dict with the detector element name and channel histogram edges.
+    """
+    # Only care about the offline channels (0) and detector element name (2) columns.
+    df = pd.read_table(DETECTOR_CHANNELMAPS[map_name], sep="\s+", usecols=[0, 2], header=None)
+    named_series = pd.Series(df[0].values, index=df[2])  # Many channels will belong to the same det_elem.
```

Comment:
Why many channels will belong to the same det element?

Reply:
Each det element is each APA in HD, so there will be 4 unique det element names that are repeated for 2560 channels each. For VD cold box, there is only one det element, so it's one name repeated for 3072 channels.